include/onnxruntime/core/framework/func_api.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 #include "core/common/status.h"
+using onnxruntime::common::Status;
```

Comment:
Shouldn't use "using" in header files.

Reply:
Hmm, well the original header assumed it was using the namespace already (you can see below where it just uses 'Status' which I didn't change)